index.html

```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
             <li><a href="#"><img src="./images/Vector (3).png"></a></li> 
           </ul>
     </section>
+    <section class="grid-container">
+       <div class="grid-1">
+         <img src="./images/Snapshoot Portfolio.png" alt="project1" class="project1">
+         <h1 class="ton">Tonic</h1>
```

Comment:
- Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.